src/main/common/helpers/paymentPlanHelper.ts

```diff
@@ -28,17 +65,44 @@ export class PaymentPlanHelper {
 
     switch (responseType) {
       case ResponseType.PART_ADMISSION:
-        return PaymentPlanHelper.createPaymentPlanFromClaimPartialAdmission(response as PartialAdmissionResponse)
+        return PaymentPlanHelper.createPaymentPlanFromClaimAdmission(response as PartialAdmissionResponse,
+          claim.claimData.amount.totalAmount())
```

Comment:
I think in part admission, it will be amount accepted and not the total amount. please check with P.O